src/System.Memory/ref/System.Memory.cs

```diff
@@ -209,6 +209,15 @@ public partial struct MemoryHandle : System.IDisposable
         public unsafe void* Pointer { get { throw null; } }
         public void Dispose() { }
     }
+    public abstract class MemoryPool<T> : IDisposable
+    {
+        public static System.Buffers.MemoryPool<T> Shared { get; }
+        public abstract System.Buffers.OwnedMemory<T> Rent(int minBufferSize=-1);
```

Comment:
We had a discussion at the API review about the default here. Did we really settle on -1? Why not have the default be 0?

Reply:
> Why not have the default be 0?

FWIW, ArrayPool.Shared.Rent(0) gives back an empty array.

Reply:
IIRC, we have settled on -1.

Reply:
It's on youtube, it was -1.

Reply:
you mean "it's on the internet"? :-)